deploy/dataproc/gprofiler_initialization_action.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -euxo pipefail
+
+GPROFILER_TOKEN=$(/usr/share/google/get_metadata_value attributes/gprofiler-token)
+readonly GPROFILER_TOKEN
+
+GPROFILER_SERVICE=$(/usr/share/google/get_metadata_value attributes/gprofiler-service)
+readonly GPROFILER_SERVICE
+
+wget https://github.com/Granulate/gprofiler/releases/latest/download/gprofiler
+sudo chmod +x gprofiler
+sudo sh -c "setsid ./gprofiler -cu --token '$GPROFILER_TOKEN' --service-name '$GPROFILER_SERVICE' > /dev/null 2>&1 &"
```

Comment:
I also think that, learning from the Fargate case, we should not redirect to /dev/null by default, as it hinders debugging which is often needed. I suggest you remove it and instead mention in a comment that it can be added if desired to avoid spamming (like we did in the Fargate section)

Reply:
What do I need to fix in the redirection?